zcash/address.go

```diff
@@ -16,8 +16,8 @@ var (
 	TestSpendingKey = [2]byte{0xAC, 0x08}
 	ProdAddress     = [2]byte{0x16, 0x9A}
 	TestAddress     = [2]byte{0x16, 0xB6}
-	ProdViewingKey  = [2]byte{0x0B, 0x1C}
-	TestViewingKey  = [2]byte{0x0B, 0x2A}
+	ProdViewingKey  = [3]byte{0xA8, 0xAB, 0xD3}
+	TestViewingKey  = [3]byte{0xA8, 0xAC, 0x0C}
```

Comment:
Let's make these slices so that you don't have to add `[:]` around. We are breaking the API anyway.